src/app/features/filter-bar/filter-bar.component.ts

```diff
@@ -0,0 +1,73 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { FilterModel } from '@features/filter-bar/interfaces/filter-model';
+import { ReferenceModel } from '@features/filter-bar/interfaces/reference-model';
+import { StateGroup } from '@features/filter-bar/interfaces/state-group';
+import { StateGroupsService } from '@features/filter-bar/services/state-groups.service';
```

Comment:
when component is in the same directory or in the same module - import with relative paths - that will help to clearer understand that it all is related to this component/module and not something shared